background.ts

```diff
@@ -521,6 +527,46 @@ Electron.ipcMain.on('factory-reset', async() => {
   Electron.app.quit();
 });
 
+Electron.ipcMain.on('get-app-version', (event) => {
+  let version = '';
+
+  if (process.env.NODE_ENV === 'production') {
+    try {
+      version = Electron.app.getVersion();
+    } catch (err) {
+      console.log(`Can't get app version: ${ err }`);
+      version = '?';
+    }
+    event.returnValue = version;
+  } else {
+    try {
+      const child = childProcess.execFile('git', ['describe', '--tags']);
```

Comment:
Using the [helper class](https://github.com/rancher-sandbox/rancher-desktop/blob/main/src/utils/childProcess.ts) might be easier.
```ts
try {
  const { stdout } = await childProcess.spawnFile('git', ['--describe', '--tags'], { stdio: ['ignore', 'pipe', 'inherit'] });
  event.returnValue = stdout.trim();
} catch (ex) {
  console.error('Could not get app version', ex);
  event.returnValue = '?';
}
```